packages/ccd-js-gen/package.json

```diff
@@ -37,6 +37,9 @@
         "url": "https://concordium.com"
     },
     "license": "Apache-2.0",
+    "engines": {
+        "node": ">=16.15.0 <21"
```

Comment:
It should warn the user, when installing the package now.
The library should actually work with more versions, it is just the CLI, which uses a feature which is new causing most of these issues... maybe I should just avoid using that feature in the first place, meaning it will work in every version...

Reply:
I have updated the PR, such that `ccd-js-gen` supports the remaining node versions, meaning there is no need to have this anymore.